tutorials/haskell/hspec-golden-test-tutorial/code/src/HelloWorld.hs

```diff
@@ -0,0 +1,6 @@
+module HelloWorld where
+
+sayHi :: String
+sayHi = "Hello Golden Testers"
+    
+    
```

Comment:
@Jagl257 Let's remove this extra white lines.

Reply:
Removed